core/src/main/scala/quasar/physical/ts/TSDestination.scala

```diff
@@ -91,30 +96,100 @@ final class TSDestination[F[_]: Concurrent: ContextShift: MonadResourceErr] priv
 
       val r = for {
         tableName <- Resource.liftF[F, String](tableNameF)
-        p <- client.exec(cc, loadCommand(tableName), blocker)
-        _ <- bytes
-          .through(gzip[F](BufferSize))
+
+        _ <- Resource.liftF(
+            Sync[F].delay(
+              log.info(s"(re)creating ${config.database}.${tableName} with schema ${columns.show}")))
+
+        p <- client.exec(cc, "tql", blocker)
```

Comment:
This is supposed to be `"tql"`? Not `"sql"`?

Reply:
Yep.